x-pack/test_serverless/api_integration/test_suites/observability/ai_assistant/tests/conversations/conversations.spec.ts

```diff
@@ -0,0 +1,291 @@
+            internalReqHeader,
+            roleAuthc,
+          })
+          .expect(200);
+
+        expect(response.body).to.eql({ conversations: [] });
+      });
+
+      it('returns a 404 for updating conversations', async () => {
+        await observabilityAIAssistantAPIClient
+          .slsUser({
+            endpoint: 'PUT /internal/observability_ai_assistant/conversation/{conversationId}',
+            internalReqHeader,
+            roleAuthc,
+            params: {
+              path: {
+                conversationId: 'non-existing-conversation-id',
+              },
+              body: {
+                conversation: conversationUpdate,
+              },
+            },
+          })
+          .expect(404);
+      });
+
+      it('returns a 404 for retrieving a conversation', async () => {
+        await observabilityAIAssistantAPIClient
+          .slsUser({
+            endpoint: 'GET /internal/observability_ai_assistant/conversation/{conversationId}',
+            internalReqHeader,
+            roleAuthc,
+            params: {
+              path: {
+                conversationId: 'my-conversation-id',
+              },
+            },
+          })
+          .expect(404);
+      });
+    });
+
+    describe('when creating a conversation with the write user', function () {
+      let createResponse: Awaited<
+        SupertestReturnType<'POST /internal/observability_ai_assistant/conversation'>
+      >;
+      before(async () => {
+        createResponse = await observabilityAIAssistantAPIClient
+          .slsUser({
+            endpoint: 'POST /internal/observability_ai_assistant/conversation',
+            roleAuthc,
+            internalReqHeader,
+            params: {
+              body: {
+                conversation: conversationCreate,
+              },
+            },
+          })
+          .expect(200);
+      });
+
+      after(async () => {
+        await observabilityAIAssistantAPIClient
+          .slsUser({
+            endpoint: 'DELETE /internal/observability_ai_assistant/conversation/{conversationId}',
+            internalReqHeader,
+            roleAuthc,
+            params: {
+              path: {
+                conversationId: createResponse.body.conversation.id,
+              },
+            },
+          })
+          .expect(200);
+
+        await observabilityAIAssistantAPIClient
+          .slsUser({
+            endpoint: 'GET /internal/observability_ai_assistant/conversation/{conversationId}',
+            internalReqHeader,
+            roleAuthc,
+            params: {
+              path: {
+                conversationId: createResponse.body.conversation.id,
+              },
+            },
+          })
+          .expect(404);
+      });
+      it('returns the conversation', function () {
+        // delete user from response to avoid comparing it as it will be different in MKI
+        delete createResponse.body.user;
```

Comment:
delete the `user` prop when checking the response.  it only has `user.name` and this is a string of numbers in MKI that can change.